src/android/app/src/main/java/org/citra/citra_emu/features/settings/model/IntSetting.kt

```diff
@@ -33,6 +33,7 @@ enum class IntSetting(
     LANDSCAPE_BOTTOM_Y("custom_bottom_y",Settings.SECTION_LAYOUT,480),
     LANDSCAPE_BOTTOM_WIDTH("custom_bottom_width",Settings.SECTION_LAYOUT,640),
     LANDSCAPE_BOTTOM_HEIGHT("custom_bottom_height",Settings.SECTION_LAYOUT,480),
+    SCREEN_GAP("screen_gap",Settings.SECTION_LAYOUT,0),
```

Comment:
If `screen_gap` is supposed to be a float, then it should be kept in `FloatSetting.kt`.

Reply:
Since it's measured in pixels it should be an int I think